.github/workflows/release-please.yml

```diff
@@ -0,0 +1,18 @@
+name: Update release PR
+
+on:
+    push:
+        branches:
+            - main
+
+permissions:
+    contents: write
+    pull-requests: write
+
+jobs:
+    release-please:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: googleapis/release-please-action@v4
+              with:
+                  token: ${{ secrets.RELEASE_PLEASE_GITHUB_TOKEN }}
```

Comment:
Half nit: the Internet isn't certain about it, but it's said YAML looks nicer with 2-space indentation.

Reply:
Yep, agreed, let me fix!

Reply:
Ok, I've fixed that using `commit --no-verify` and fixed the underlying issue [here](https://github.com/Flagsmith/flagsmith-engine/pull/216).